### PR TITLE
Add function of binding zapper id with user

### PR DESCRIPTION
**Pull request by winonecheng**

### CODE REVIEW

File: denguefever_tw/dengue_linebot/templates/dengue_linebot/bot_templates/zh_tw/_functions.j2 [hunk 1]
```diff
@@ -5,3 +5,4 @@
 5. 即時疫情
 6. 給予建議 or 想對我們說什麼？
 7. 登錄位置
+8. 綁定補蚊燈(beta)
```
Comment (Lee-W) [inline denguefever_tw/dengue_linebot/templates/dengue_linebot/bot_templates/zh_tw/_functions.j2]: You should add the end of line at the last line of the file.

i.e.
You should
```
....
....\n
```

But not
```
...
...\n
\n
```
or
```
...
...
```

File: denguefever_tw/dengue_linebot/templates/dengue_linebot/bot_templates/zh_tw/bind_zapper.j2 [hunk 2]
```diff
@@ -0,0 +1 @@
+將貼在補蚊燈上的id告訴我，就能綁定補蚊燈囉！
```
Comment (Lee-W) [inline denguefever_tw/dengue_linebot/templates/dengue_linebot/bot_templates/zh_tw/bind_zapper.j2]: Same above

File: denguefever_tw/static/dengue_linebot/config/FSM.json [hunk 3]
```diff
@@ -82,7 +82,16 @@
         "unrecognized_msg",
         "user_register_location",
         "wait_register_location",
-        "receive_register_location"
+        "receive_register_location",
+        {
+            "name": "wait_bind_zapper",
+            "on_enter": {
+                "name": "on_enter_wait_bind_zapper",
```
Comment (Lee-W) [inline denguefever_tw/static/dengue_linebot/config/FSM.json:89]: You haven't implemented these methods, am I correct?

File: denguefever_tw/dengue_linebot/templates/dengue_linebot/bot_templates/zh_tw/bind_zapper.j2 (hunk 2 above)
Comment (winonecheng) [reply denguefever_tw/dengue_linebot/templates/dengue_linebot/bot_templates/zh_tw/bind_zapper.j2]: Fixed

File: denguefever_tw/dengue_linebot/templates/dengue_linebot/bot_templates/zh_tw/_functions.j2 (hunk 1 above)
Comment (winonecheng) [reply denguefever_tw/dengue_linebot/templates/dengue_linebot/bot_templates/zh_tw/_functions.j2]: Fixed

**Review by winonecheng — COMMENTED** [review]
Ignore this comment...

File: denguefever_tw/static/dengue_linebot/config/FSM.json (hunk 3 above)
Comment (winonecheng) [reply denguefever_tw/static/dengue_linebot/config/FSM.json:89]: Finished in next commit

**Comment by winonecheng** [comment]
Since the zapper id list is not available yet, so I haven't implemented id format check.

File: denguefever_tw/static/dengue_linebot/config/FSM.json (hunk 3 above)
Comment (Lee-W) [reply denguefever_tw/static/dengue_linebot/config/FSM.json:89]: Ok

**Comment by Lee-W** [comment]
Ok

**Review by Lee-W — APPROVED** [review]
LGTM. Will merge.

**Pull request merged**